libs/s25main/desktops/Desktop.cpp

```diff
@@ -26,11 +26,7 @@ Desktop::Desktop(glArchivItem_Bitmap* background)
 {
     SetScale(true);
     SetFpsDisplay(true);
-    // By default limit the maximum frame rate to 60 FPS
-    if(SETTINGS.video.vsync < 0)
-        VIDEODRIVER.setTargetFramerate(60);
-    else
-        VIDEODRIVER.setTargetFramerate(SETTINGS.video.vsync);
+    VIDEODRIVER.setTargetFramerate(SETTINGS.video.framerate);
```

Comment:
This is intentional: For all non-ingame purposes "unlimitted" should mean "60" as otherwise you'll get insane framerates for no benefit.
So please revert this part of the change

Reply:
Oh, you are right, I misunderstood the point of this code

Reply:
reverted